sdks/java/io/elasticsearch/contrib/logstash.conf

```diff
@@ -0,0 +1,61 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+# Inject documents containing scientists among 'Einstein', 'Darwin', 'Copernicus', 'Pasteur', 'Curie', 'Faraday', 'Newton', 'Bohr',
+# 'Galilei', 'Maxwell' into elasticsearch in an index called beam using a doctype called test
+input {
+  generator {
+    codec => "json"
+    threads => 4
+    lines => [
+      '{
+          "scientist": "name"
+      }'
+    ]
+    # count is multiplied by threads (!)
+    # By default ES will create 5 shards, so beam IO will create 5 sources.
+    # Inject 60 000 docs so that each source reads more docs than ES read window (10 000)
```

Comment:
Well, we had problems on high volume (hence integration tests) before implementing the IO with scroll. Indeed with standard from/size reading, ES uses a window of 10 000 documents by default. So from and size parameters cannot be more than 10 000. That is why we chose to use scroll instead. Scroll do not use ES window. So there should not be any problem about that. Nevertheless I wanted to test with a lot of documents for each source so that we detect any problem with high volume. I will remove the comment about window (because of the use of scroll) and only explain about the high volume.